### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by ManojKRISHNAPPA**
this is based latest code bug we fixed

### CODE REVIEW

File: controller.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <label for="fname"><i class="fa fa-user"></i> Full Name</label>
+            <input type="text" id="fname" name="firstname" placeholder="John M. Doe">
+            <label for="email"><i class="fa fa-envelope"></i> Email</label>
+            <input type="text" id="email" name="email" placeholder="john@example.com">
+            <label for="adr"><i class="fa fa-address-card-o"></i> Address</label>
+            <input type="text" id="adr" name="address" placeholder="542 W. 15th Street">
+            <label for="city"><i class="fa fa-institution"></i> City</label>
+            <input type="text" id="city" name="city" placeholder="New York">
+
+            <div class="row">
```
Comment (nandishkumar) [inline controller.java:18]: This logic is wrong

File: controller.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                <input type="text" id="state" name="state" placeholder="NY">
+              </div>
+              <div class="col-50">
+                <label for="zip">Zip</label>
+                <input type="text" id="zip" name="zip" placeholder="10001">
+              </div>
+            </div>
+          </div>
+
+          <div class="col-50">
```
Comment (nandishkumar) [inline controller.java:30]: This looks fine

**Review by nandishkumar — COMMENTED** [review]
Commit,check it

**Comment by nandishkumar** [comment]
Testing

**Review by ManojKRISHNAPPA — COMMENTED** [review]
test

File: controller.java (hunk 1 above)
Comment (ManojKRISHNAPPA) [reply controller.java:18]: logic is corrected

**Pull request merged**